tests/test_api_client.py

```diff
@@ -0,0 +1,55 @@
+import sys
+import pytest
+import requests
+from pathlib import Path
+from unittest.mock import Mock, patch
+from dataclasses import dataclass
+
+sys.path.append(str(Path(__file__).resolve().parent.parent))
+from reaskapi.api_client import ClientConfig
+from reaskapi.deepcyc import DeepCyc
+from reaskapi.metryc import Metryc
+
+
+@dataclass
+class MockedResponse:
+    status_code: int = 200
+
+    def json(arg):
+        return {}
+
+
+@pytest.mark.parametrize("config", [None, ClientConfig("https://localhost:8001")])
```

Comment:
Could you give me a bit more details about this comment?
The tests in here are to verify if DeepCyc and Metry are initialized with ClientConfig.base_url accordingly and use the base_url by requests from these classes.

Reply:
I see, it would not make sense to have the Metryc and Deepcyc objects as parameters to a single test_config() function. 

However we could have something like:

@pytest.mark.parametrize("product", ["metryc", "deepcyc")])
def test_config(...)

And then within the function:

if product == 'metryc':
   obj = Metryc(config=config)
else:
   obj = DeepCyc(config=config)

obj.tctrack_events()

Reply:
It makes sense. I updated the PR.